pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -820,16 +821,32 @@ func ensureCertificate(kubeClient client.Client, caPEM []byte, privateKey []byte
 	}
 }
 
-func getUserCACert(kubeClient client.Client) (string, error) {
+func getConfigMap(kubeClient client.Client, namespace, name string) (*v1.ConfigMap, error) {
 	cm := &v1.ConfigMap{
 		TypeMeta:   metav1.TypeMeta{APIVersion: "v1", Kind: "ConfigMap"},
-		ObjectMeta: metav1.ObjectMeta{Name: UserCABundleConfigMap},
+		ObjectMeta: metav1.ObjectMeta{Name: name},
+	}
+	err := kubeClient.Get(context.TODO(), client.ObjectKey{Namespace: namespace, Name: name}, cm)
+	if err != nil {
+		return nil, err
+	}
+	return cm, nil
+}
+
+func getUserCACert(kubeClient client.Client) (string, error) {
+	cm, err := getConfigMap(kubeClient, OpenShiftConfigNamespace, UserCABundleConfigMap)
+	if err != nil {
+		return "", err
 	}
-	err := kubeClient.Get(context.TODO(), client.ObjectKey{Namespace: OpenShiftConfigNamespace, Name: UserCABundleConfigMap}, cm)
+	return cm.Data["ca-bundle.pem"], nil
```

Comment:
before we were doing string(cm.Data...), was that not needed?

Reply:
Nah, it's `String` already. I copied it from the other operators code but my editor warned that conversion is redundant.